todo_app/data/session_items.py

```diff
@@ -30,7 +30,7 @@ def get_item(id):
     return next((item for item in items if item['id'] == int(id)), None)
 
 
-def add_item(title):
+def add_item(item):
```

Comment:
Here you've changed the name of this parameter, but not where  it's used. This leads to errors when this function is called, because `title` isn't defined in line 48
Leaving this named `title` is probably clearer here, but alternatively changing line 48 to use the new `item` variable name as below would also work:
```
item = { 'id': id, 'title': item, 'status': 'Not Started' }
```